hypervisor/debug/profiling.c

```diff
@@ -3,19 +3,19 @@
  *
  * SPDX-License-Identifier: BSD-3-Clause
  */
-
 #ifdef PROFILING_ON
 
 #include <hypervisor.h>
-#include <acrn_common.h>
-#include <hv_arch.h>
-#include <hv_debug.h>
-#include <profiling.h>
 
 #define ACRN_DBG_PROFILING		5U
 
+#define MAJOR_VERSION			1
+#define MINOR_VERSION			0
```

Comment:
use 1U, 0U to comply with MISRO C

Reply:
In my implementation later I am assigning these macro's into an int32_t type. 
Is MACRO always supposed to be unsigned ? If it isn't the case it would introduce extra casting at the time of assignment.

Reply:
this one is fine